deploy-as-code/helm/charts/works/backend/attendance/values.yaml

```diff
@@ -19,6 +19,46 @@ initContainers:
     schemaTable: "attendance_schema"
     image:
       repository: "attendance-db"
+    env: |
+      {{- if index .Values "attendance-ismultischema-enabled" }}        
```

Comment:
If the services are deployed into staging namespace then db migration is not required. Only if deploy in the egov namespace is deployed then required.